### PR TITLE
Update development-process.md

### PR DESCRIPTION
**Pull request by Tanmay000009**
Github Bug/Issue: Fixes https://github.com/taskcluster/taskcluster/issues/4947#issuecomment-908614772

Added two snippets under: https://github.com/taskcluster/taskcluster/blob/main/dev-docs/development-process.md#hacking-on-the-ui under 
- Set both READ_DB_URL and WRITE_DB_URL to the URL for your postgres server, as described above.

Reference: https://github.com/taskcluster/taskcluster/issues/2637#issuecomment-617997973 

Also added a note about setting all environment variables in the same shell session.

### CODE REVIEW

File: dev-docs/development-process.md [hunk 1]
```diff
@@ -89,6 +89,14 @@ To run the Taskcluster UI:
 * In a shell window for taskcluster-web-server:
   * Make sure Postgres is set up as described above.
   * Set both `READ_DB_URL` and `WRITE_DB_URL` to the URL for your postgres server, as described above.
+    For example:
+    ```sh
+    export READ_DB_URL=$TEST_DB_URL
+    ```
+    ```sh
+    export WRITE_DB_URL=$TEST_DB_URL
+    ```
```
Comment (jwhitlock) [inline dev-docs/development-process.md:98]: Let's combine the two into a single block:

```suggestion
    ```sh
    export READ_DB_URL=$TEST_DB_URL
    export WRITE_DB_URL=$TEST_DB_URL
    ```
```

File: dev-docs/development-process.md [hunk 2]
```diff
@@ -106,6 +114,8 @@ To run the Taskcluster UI:
     You will only need to do this when the dependencies change.
   * Run `yarn start` to start the development server.
     It will output a localhost URL on which you can see the site.
+    
+    *Note* : Make sure you have set all the above mentioned environment variables in the shell session.
```
Comment (jwhitlock) [inline dev-docs/development-process.md:118]: This note should be in the previous section. The commands run in `services/web-server` need the environment variables, while the ones run in `ui` do not.

There's already a note in that block, you may want to format both as sentences, like 

> "If you get an error like `readDbUrl is required`, ensure you've set all the environment variables in the shell session."

or both notes like this new style:

> *Note 1*: The pulse note
> *Note 2*: The environment note

**Review by jwhitlock — COMMENTED** [review]
Thanks @Tanmay000009! I think the sample command improves the docs. The note about setting environment variables is in the wrong section, can be moved up.

File: dev-docs/development-process.md (hunk 2 above)
Comment (Tanmay000009) [reply dev-docs/development-process.md:116]: Sure got it!

**Comment by Tanmay000009** [comment]
I have got the changes, and will make a clean PR with changelogs aswell!

**Pull request closed without merging**